### DISCUSSION TITLE
Chat stops after upgrade 4.29

### DISCUSSION THREAD

**Discussion by Broninn** [LiveHelperChat/livehelperchat#2015] [Q&A]
Hello everyone, I just updated the LHC version to 4.29, and now the chat has stopped working.

![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/940d97be-c347-4f8c-b9aa-e6d002688fd9)
![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/8e505f86-7f68-4584-8819-b847f8399117)
![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/5e252c40-794e-4967-83a0-fab8af73cc2a)

**Comment by Broninn** [comment]
There is no service waiting on the admin screen, but I already have a queue waiting

**Comment by Broninn** [comment]
Update: the list of pending chats does not appear for operators. For administrators, the pending chat list appears

**Reply by remdex** [reply]
please see documentation for chat list. There is new permissions you have to grant to your agents..

**Reply by remdex** [reply]
https://doc.livehelperchat.com/docs/dashboard#required-permissions-for-chat-listing
